exampleCourse/questions/downloadFile/question.html

```diff
@@ -0,0 +1,8 @@
+<pl_question_panel>
+    <ul>
+        <li>This text file comes from <code>clientFilesCourse</code>: <pl_file_download file_name="data.txt" type="static" scope="course" /></li>
```

Comment:
I'm not opposed to this. How will it work with "dynamic" files? One option is to eliminate `type` and `scope` in favor of `directory` as you say, and to indicate a dynamic file with `generatedFilesQuestion` - that seems cryptic. Another option is to keep things the way they are, with `generatedFilesQuestion` being the default value of `directory` in the case of `type=dynamic`.